modules/scip/9.2.0.bcr.1/patches/scip.patch

```diff
@@ -0,0 +1,187 @@
++#define WITH_SCIPDEF
++/* #undef SCIP_WITH_LAPACK */
++/* #undef SCIP_WITH_PAPILO */
++#define SCIP_WITH_ZLIB
++/* #undef SCIP_WITH_READLINE */
++/* #undef SCIP_WITH_GMP */
++/* #undef SCIP_WITH_LPSCHECK */
++/* #undef SCIP_WITH_ZIMPL */
++/* #undef SCIP_WITH_AMPL */
++#define SCIP_ROUNDING_FE
++/* #undef SCIP_ROUNDING_FP */
++/* #undef SCIP_ROUNDING_MS */
++
++#endif
+diff --git a/src/scip/scip_export.h b/src/scip/scip_export.h
+new file mode 100644
+index 0000000000..871fde8e55
+--- /dev/null
++++ b/src/scip/scip_export.h
+@@ -0,0 +1,24 @@
++
++#ifndef SCIP_EXPORT_H
++#define SCIP_EXPORT_H
++
++#ifdef SCIP_STATIC_DEFINE
++#  define SCIP_EXPORT
++#  define SCIP_NO_EXPORT
++#else
++#  ifndef SCIP_EXPORT
++#    ifdef libscip_EXPORTS
++/* We are building this library */
++#      define SCIP_EXPORT
++#    else
++/* We are using this library */
++#      define SCIP_EXPORT
++#    endif
++#  endif
++
++#  ifndef SCIP_NO_EXPORT
++#    define SCIP_NO_EXPORT
++#  endif
++#endif
++
++#endif /* SCIP_EXPORT_H */
+diff --git a/src/scip/githash.c b/src/scip/githash.c
+new file mode 100644
+index 0000000000..d1e99c662d
+--- /dev/null
++++ b/src/scip/githash.c
+@@ -0,0 +1 @@
++#define SCIP_GITHASH "a740f0891e"
+\ No newline at end of file
+diff --git a/src/scip/scipbuildflags.c b/src/scip/scipbuildflags.c
+--- a/src/scip/scipbuildflags.c
++++ b/src/scip/scipbuildflags.c
+@@ -23,7 +23,6 @@
+ /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+ /**@file   scipbuildflags.c
+- * @ingroup OTHER_CFILES
+  * @brief  build flags methods
+  * @author Felipe Serrano
+  */
+@@ -31,12 +30,36 @@
+ /*---+----1----+----2----+----3----+----4----+----5----+----6----+----7----+----8----+----9----+----0----+----1----+----2*/
+
+ #include "scip/scipbuildflags.h"
+-#include "scip/buildflags.h"
+
+ /** returns the flags that were used to build SCIP */
+ const char* SCIPgetBuildFlags(
+    void
+    )
+ {
+-   return SCIP_BUILDFLAGS "\n LPS=" SCIP_LPS "\n IPOPT=" SCIP_IPOPT;
++   return " ARCH=arm64\n\
++ OSTYPE=Darwin-23.6.0\n\
++ COMP=AppleClang 16.0.0.16000026\n\
++ BUILD=Release\n\
++ DEBUGSOL=OFF\n\
++ EXPRINT=none\n\
++ SYM=none\n\
++ GMP=OFF\n\
++ IPOPT=OFF\n\
++ WORHP=OFF\n\
++ LPS=none\n\
++ LPSCHECK=OFF\n\
++ NOBLKBUFMEM=OFF\n\
++ NOBLKMEM=OFF\n\
++ NOBUFMEM=OFF\n\
++ THREADSAFE=ON;FORCE\n\
++ READLINE=OFF\n\
++ SANITIZE_ADDRESS=OFF\n\
++ SANITIZE_MEMORY=OFF\n\
++ SANITIZE_UNDEFINED=OFF\n\
++ SANITIZE_THREAD=OFF\n\
++ SHARED=OFF\n\
++ VERSION=9.2.0.0\n\
++ API_VERSION=115\n\
++ ZIMPL=OFF\n\
++ ZLIB=ON";
+ }
+diff --git a/src/symmetry/compute_symmetry_bliss.cpp b/src/symmetry/compute_symmetry_bliss.cpp
+index 26117a8026..2f164d133b 100644
+--- a/src/compute_symmetry_bliss.cpp	2024-11-08 16:36:13
++++ b/src/symmetry/compute_symmetry_bliss.cpp	2024-11-08 16:37:32
+@@ -34,8 +34,9 @@
+ #include "compute_symmetry.h"
+
+ /* include bliss graph */
+-#include <bliss/defs.hh>
+-#include <bliss/graph.hh>
++#define BLISS_VERSION "0.73"
```

Comment:
Can we get this define in the bliss package defs.h or similar?

Reply:
maybe for another time. please